src/main/java/com/example/onlinebookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,8 @@
+package com.example.onlinebookstore.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
Why this exception is checked?

Reply:
Becouse 2 reasons:
1. App still need to be running, after exception occures
2. Its used to handle errors, coused by user input.

Leave unchanged.